CleanerRobot.md

```diff
@@ -0,0 +1,14 @@
+Robot Name: Cleaner Robot
+
+Function: Sweep Floor Really Well
```

Comment:
This and all of the other functions should be renamed as "tasks".

Reply:
changed